src/gcp_scanner/arguments.py

```diff
@@ -143,5 +145,8 @@ def arg_parser():
         'Please select at least one option to begin scan\
  -k/--sa-key-path,-g/--gcloud-profile-path, -m, -rt, -at'
     )
+  if not os.path.exists(args.output):
+    logging.error("%s doesn't exists. Please enter a valid path.", args.output)
```

Comment:
Add quotes and fix grammar: `"\"%s\" doesn't exist. Please enter a valid directory path."`